lib/src/container/store.rs

```diff
@@ -355,6 +369,46 @@ pub fn list_images(repo: &ostree::Repo) -> Result<Vec<String>> {
         .collect()
 }
 
+fn query_image_impl(
+    repo: &ostree::Repo,
+    imgref: &OstreeImageReference,
+) -> Result<Option<(ImageManifest, LayeredImageState)>> {
```

Comment:
I thought about exposing the parsed manifest in `LayeredImageState` too, but it can come later.